letsencrypt/tests/colored_logging_test.py

```diff
@@ -0,0 +1,39 @@
+"""Tests for letsencrypt.colored_logging."""
+import logging
+import StringIO
+import unittest
+
+from letsencrypt import le_util
+
+
+class StreamHandlerTest(unittest.TestCase):
```

Comment:
following rest of the code we should have `"""Tests for letsencrypt.colored_logging.StreamHandler."""` here - it's nice for grepping in some situations